src/components/LateNight/LateNightAllow/LateNightAllowItem/index.tsx

```diff
@@ -16,6 +19,13 @@ const LateNightAllowItem = ({
   const { data: lateNightAllow } = useGetPendingLateNight();
   const { handleLateNightAllow, patchLateNightAllow, patchLateNightCancel } =
     useLateNightAllow();
+  const [isOpen, setIsOpen] = useState(false);
+  const [studyData, setStudyData] = useState<LateNightType>();
+
+  const handleModalClick = () => {
+    setIsOpen(!isOpen);
```

Comment:
```suggestion
    setIsOpen(true);
```
명시적으로 보여주는게 나아보입니다 !isOpen은 useState에 초기값을 봐아 알 수 있기 때문에요

Reply:
수정했습니다 :)